tests/unit/test-class-wc-payments.php

```diff
@@ -167,6 +167,41 @@ function () {
 		$this->assertNotEquals( 'woocommerce_rest_missing_nonce', $response->get_data()['code'] );
 	}
 
+	public function test_temp() {
+		// Necessary in order to prevent wp_die() from being called.
```

Comment:
This was my typo. I didn't notice this yesterday but I meant to say the following:
```
// Necessary in order to prevent die from being called.
```

Since defining `DOING_AJAX` to `true` forces `wp_die()` to be called.